sycl/test/graph/graph-explicit-dotp.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <CL/sycl.hpp>
+
+#include <sycl/ext/oneapi/experimental/graph.hpp>
+
+const size_t n = 10;
+
+float host_gold_result() {
```

Comment:
This reference function isn't called in the test

Reply:
It is called at the end of the test for simple results verification, though this was a recent change possibly after this comment was made?